### PR TITLE
Update to Dart 2.0's `webdev build` instead of `pub build`

### PR DESCRIPTION
**Pull request by jifalops**
There are three changes here. The third one might not be strictly necessary, but I noticed it was adding a lot of git files from a pubspec dependency defined via `path`.

1. Use `webdev build` instead of `pub build`.
2. Make the offline URLs list raw strings (fix for `$sdk` dir name).
3. Add .git/ files to the exclusion globs.

### CODE REVIEW

**Comment by jifalops** [comment]
I'm not sure if this is something I did since I never got the original version working, but offline mode does not work.

**Comment by isoos** [comment]
Oh, I'm sorry, I've missed this PR in August. @jifalops can you please revisit and see if this needs any updates since that changes?

**Comment by jifalops** [comment]
I don't see any changes in the master branch that would stop this from working. It isn't much more than a careful search and replace of `pub build` with `webdev build`. I changed the default build directory from `build/web/` to just `build/`. ~~I can't remember why exactly but there must have been some reason.~~ webdev outputs to build/, not build/web/.

**Comment by Bohne13** [comment]
I hope this pull request will be released soon

**Comment by isoos** [comment]
@kevmoo I'm way out of touch with the current webdev space, could you please review this?

File: bin/pwa.dart [hunk 1]
```diff
@@ -103,24 +103,24 @@ Future _generateManifestJson(String webDir, Map pubspec) async {
...
       if (lastSourceTimestamp < lastOfflineTimestamp) return;
     }
-    print('Running pub build:');
-    String executable = 'pub';
+    print('Running webdev build:');
+    String executable = 'webdev';
     if (Platform.isWindows) {
       try {
-        final pr = await Process.run('pub.exe', ['--version']);
+        final pr = await Process.run('webdev.exe', ['--version']);
```
Comment (devkabiir) [inline bin/pwa.dart:121]: `.exe` doesn't make sense here, If targeting windows it should be `webdev.bat` or simply `webdev` to let the platform figure out.

**Comment by isoos** [comment]
Closing as @muniz95 fixed it for 0.2.0

**Pull request closed without merging**